tools/xcodeproj-shims/xcodeproj-installer.sh

```diff
@@ -0,0 +1,26 @@
+# Copies the xcodeproject from the bazel output directory to the BAZEL_WORKSPACE directory when ran
+set -eu
+readonly project_path="${PWD}/$(project_short_path)"
+readonly dest="${BUILD_WORKSPACE_DIRECTORY}/$(project_short_path)/"
+readonly tmp_dest=$(mktemp -d)/$(project_full_path)/
+
+readonly stubs_dir="${dest}/bazelstubs"
+mkdir -p ${stubs_dir}
+
+readonly installer="$(installer_short_path)"
+
+mkdir -p $(dirname "${stubs_dir}/${installer}")
+cp "${installer}" "${stubs_dir}/${installer}"
+cp "$(clang_stub_short_path)" "${stubs_dir}/clang-stub"
+cp "$(clang_stub_ld_path)" "${stubs_dir}/ld-stub"
+cp "$(clang_stub_swiftc_path)" "${stubs_dir}/swiftc-stub"
+
+rm -fr ${tmp_dest}
```

Comment:
```suggestion
rm -fr "${tmp_dest}"
```

to prevent spaces in directories from splitting args